src/__tests__/articleTests/createArticleReducer.test.js

```diff
@@ -19,4 +19,25 @@ describe("create article reducers", () => {
     };
     expect(articlesReducer({}, startAction)).toEqual({});
   });
+  // it("should handle view articles", () => {
+  //   const startAction = {
+  //     type: ACTION_TYPES.VIEW_ARTICLES
+  //   };
+  //   const initialState = {
+  //     count: 0,
+  //     next: null,
+  //     previous: null,
+  //     article: {},
+  //     articles: [],
+  //     results: []
+  //     // onearticle: { read_time: "00:00:00" }
+  //   };
+  //   expect(articlesReducer(initialState, startAction)).toEqual({});
```

Comment:
Remove all commented code @kcharles52

Reply:
I have removed them. I was trying to help write the tests for Amos